### DISCUSSION TITLE
Web part to filter planner plans based on the Assigned To user?

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3785] [Q&A]
We have a Teams which contain around 40++ planner plans. The managers are facing troubles when they want to see all the tasks from all the planner plans assigned to certain user. Currently to do this, they need to go to each individual plan and do the filter, So can we for example benefit from modern search web part to achieve this? or any other web parts?

I tried the following "Query template" and the following "Entity type", but i did not get any result:-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/d295e00e-992e-4967-88e4-35044bb9ec70)

Any advice?

**Comment by wobba** [comment]
Planner is not indexed in SharePoint nor Microsoft Search, so not searchable via these API's.

**Comment by mvcsharepointdev** [comment]
@wobba thanks for the reply. so what we can do to get Tasks inside web parts?

**Reply by wobba** [reply]
Create or find a custom web part using API's against planner.

**Comment by kasperbolarsen** [comment]
As Planner Tasks are not indexed PnP Modern Search web parts are not the solution.  You might look into creating a SPFx web part using Graph to access tasks. A quick search on the web seems to indicate that such a web part exists already,  both as commercial and OSS
